lib/feature_flagger/model.rb

```diff
@@ -18,16 +18,16 @@ def rollout?(*feature_key)
       FeatureFlagger.control.rollout?(feature.key, id)
     end
 
-    def release!(*feature_key)
+    def release(*feature_key)
```

Comment:
The deprecated method should be on `Model` instead of `Control` or on both of it since the actual use of the gem is with the `Model` methods:

```ruby
account.release!(:email_marketing, :new_flow)
```